jsx/StaticDataTable.js

```diff
@@ -324,6 +324,20 @@ var StaticDataTable = React.createClass({
     var rows = [];
     var curRow = [];
     var index = this.getSortedRows();
+
+    // Start Imaging Browser 17.1 Quickfix
+    if (window.location.pathname.indexOf('imaging_browser') > -1) {
+      var sortedSessIDs = [];
+      var sessIndex = this.props.Headers.indexOf('SessionID');
+      for (var i = 0; i < index.length; i++) {
+        sortedSessIDs.push(
+          this.props.Data[index[i].RowIdx][sessIndex]
+        );
+      }
+      var url = loris.BaseURL + '/imaging_browser/ajax/setSortedRows.php';
+      $.post(url, {sortedIDs: sortedSessIDs});
+    }
+    // End Imaging Browser 17.1 Quickfix
```

Comment:
This doesn't make sense to add to render method of the static data table class because:

1. It's a huge hack to add special cases for different modules, and defeats the purpose of having a general class in the first place.
2. The render function can't have side-effects like making a post request. It needs to only render content based on the state and props.

Why can't this go into the imaging browser module itself? If the Static/DynamicDataTable needs to be modified, it should be to extend the number of places that components can hook into it (ie. by having and onSort or onBlur property added which triggers a callback at the appropriate time, not by leaking module specific exceptions outside of the module itself.)